README.md

```diff
@@ -25,4 +25,18 @@ A Vagrantfile and puppet manifest are available for development within a virtual
   * [Vagrant](https://docs.vagrantup.com)
   * [VirtualBox](https://www.virtualbox.org)
 
-To load and enter the VM: `vagrant up && vagrant ssh`
\ No newline at end of file
+To load and enter the VM: `vagrant up && vagrant ssh`
+
+## database migrations
```

Comment:
I'd suggest adding the thought that the following discussion is relevant only if the microservice makes use of a database. Otherwise it may seem too complex and scare off a new developer. Not sure.